blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/CoreConfig.java

```diff
@@ -141,7 +141,8 @@
     STRING_METRICS_DROPPED("false"),
     TENANTIDS_TO_KEEP(""),
 
-    USE_ES_FOR_UNITS("false");
+    USE_ES_FOR_UNITS("false"),
+    ES_UNIT_THREADS("50"); // Should at least be equal to the number of the netty worker threads
 
```

Comment:
what about the other way around, could we define the HttpConfig in terms of this new config, ES_UNIT_THREADS()?